app/views/groups/index.html.haml

```diff
@@ -45,3 +47,4 @@
       = t :groups_no_groups
 
 = render partial: "shared/footer"
+
```

Comment:
it looks like you are adding an second empty line to the end of the file which is incorrect. should be only one empty line. can you double check please?